src/Targets/Microsoft.Managed.DesignTime.targets

```diff
@@ -21,6 +21,8 @@
 
     <!--This property sets the default output types supported by the project system.-->
     <SupportedOutputTypes Condition="'$(SupportedOutputTypes)' == ''">Exe;WinExe;Library</SupportedOutputTypes>
+
+    <SuppressOutOfDateMessageOnBuild>true</SuppressOutOfDateMessageOnBuild>
```

Comment:
It's a IVsHierarchy property (VSHPROPID_SuppressOutOfDateMessageOnBuild) - flavors can override it.